README.md

```diff
@@ -34,146 +34,123 @@ Please help us by [Contributing](CONTRIBUTING.md) to the project.
-   ./rook object user create rook-user "A rook rgw User"
-   ```
+However, before we proceed, we need to specify the Ceph monitor IPs. You can find them by running this line (you will need `jq`).
 
-3. Get the connection information for accessing object storage
+```
+$ kubectl -n rook get pod mon0 mon1 mon2 -o json|jq .items[].status.podIP
+"10.2.1.68"
+"10.2.2.66"
+"10.2.0.41"
+``` 
 
-   ```bash
-   eval $(./rook object connection rook-user --format env-var)
-   ```
+Add these IPs into the `monitors` param of the `rook-storageclass.yaml`. Also append the port `6790` to each IPs. They need to be comma-separated.
 
-4. Use an S3 compatible client to create a bucket in the object store
+Create Rook Storage class:
 
-   ```bash
-   s3cmd mb --no-ssl --host=${AWS_ENDPOINT} --host-bucket=  s3://rookbucket
-   ```
+```
+$ kubectl create -f rook-storageclass.yaml
+```
 
-5. List all buckets in the object store
+#### Run sample app
 
-   ```bash
-   s3cmd ls --no-ssl --host=${AWS_ENDPOINT} --host-bucket=
-   ```
+Now that rook is running and integrated with Kubernetes, we can create a sample app to verify it. We will create the classic wordpress and mysql apps.
+Both these apps will make use of block volumes provisioned by rook.
 
-6. Upload a file to the newly created bucket
+Start mysql and wordpress:
 
-   ```bash
-   echo "Hello Rook!" > /tmp/rookObj
-   s3cmd put /tmp/rookObj --no-ssl --host=${AWS_ENDPOINT} --host-bucket=  s3://rookbucket
-   ```
+```
+$ kubectl create -f mysql.yaml
+$ kubectl create -f wordpress.yaml
+```
 
-7. Download and verify the file from the bucket
+Both of these apps create a block volume and mount it to their respective pod. You can see the Kubernetes volume claims by running the following:
 
-   ```bash
-   s3cmd get s3://rookbucket/rookObj /tmp/rookObj-download --no-ssl --host=${AWS_ENDPOINT} --host-bucket=
-   cat /tmp/rookObj-download
-   ```
+```
+$ kubectl get pvc
+NAME             STATUS    VOLUME                                     CAPACITY   ACCESSMODES   AGE
+mysql-pv-claim   Bound     pvc-95402dbc-efc0-11e6-bc9a-0cc47a3459ee   20Gi       RWO           1m
+wp-pv-claim      Bound     pvc-39e43169-efc1-11e6-bc9a-0cc47a3459ee   20Gi       RWO           1m
+```
 
-### Kubernetes
+Get the cluster IP of the wordpress app and enter it in your brower:
 
-To run a Kubernetes cluster with Rook for persistent storage go [here](https://github.com/rook/coreos-kubernetes)
+```
+$ kubectl get svc wordpress
+NAME        CLUSTER-IP   EXTERNAL-IP   PORT(S)        AGE
+wordpress   10.3.0.155   <pending>     80:30841/TCP   2m
+```
 
-### CoreOS
+You should see the wordpress app running.
 
-Rook is also easy to run on CoreOS either directly on the host or via rkt.
+## Rook Standalone Service
 
-```
-cd demo/vagrant
-vagrant up
-```
+Rook can also be deployed as a standalone service on any modern Linux host. Refer [here](https://github.com/kokhang/rook/tree/update-readme/demo/README.md) for steps on how to run Rook on a Linux host.
```

Comment:
i think the readme is definitely missing out on showcasing a super **easy** way to use `rook` by removing the old quickstart from the front page readme.  it would be nice to get that back on the front page somehow, because it really is just that. a **quick** start.

coupled with Travis's comments about moving the k8s pre-reqs off the front page, i think that would be a good combo.